app/models/user.js

```diff
@@ -29,10 +29,10 @@ var UserSchema = new Schema({
 /**
  * Virtuals
  */
-UserSchema.virtual('password').set(function(password) {
+UserSchema.virtual('password').set( function (password) {
```

Comment:
Thank you for the feedback. Couldnt use arrow functions because of scope issues, the value of the keyword 'this' becomes undefined with arrow functions